crates/turborepo-lib/src/run/builder.rs

```diff
@@ -55,6 +55,7 @@ pub struct RunBuilder {
     api_auth: Option<APIAuth>,
     repo_root: AbsoluteSystemPathBuf,
     ui: UI,
+    daemon: Option<bool>,
```

Comment:
Could we keep this in `Opts` but use the config value to set `daemon`? That way we can keep using `Opts` as the single source of truth

Reply:
looks like we're all set here! see https://github.com/vercel/turbo/pull/8728/commits/60e9a54dd1c002583d78f6b9f5dd02b2797e541e